MultipazGettingStartedSample/composeApp/build.gradle.kts

```diff
@@ -42,6 +46,7 @@ kotlin {
             implementation(compose.foundation)
             implementation(compose.material3)
             implementation(compose.ui)
+            implementation(compose.animation)
```

Comment:
this is added to androidMain block. if this requiired for iOS? shouldn't this be added to iosMain block? is this a mistake?